lib/presentation/view/movies_grid_view.dart

```diff
@@ -34,13 +34,16 @@ class _MoviesGridViewState extends State<MoviesGridView> {
 
   @override
   void initState() {
-    widget.bloc.getMovies(widget.endpoint);
+    widget.endpoint == null
+        ? widget.bloc.getFavoriteMovies()
+        : widget.bloc.getMovies(widget.endpoint!);
```

Comment:
Why using the null case for matching with the favorites, instead of creating another value in the `Endpoints` enum for the favorites? That'd be easier to read and sustain

Reply:
refactor done!